### ISSUE TITLE
Building the engine with IceT support breaks in-situ support in parallel.

### ISSUE THREAD

**Issue by aowen87** [aowen87/BAR#1991]
Matt O'Brien was unable to use libsim in parallel at LC. When I examined the logs, libsim was unable to load the engine runtime because of an unresolved dependency on IceT and the IceT network manager class. We need to put the IceT network manager into the engine runtime library instead of in the engine_par executable so in-situ will work in parallel when IceT support is enabled.

Calling dlopen(/usr/gapps/visit/2.1.1/linux-x86_64/lib/libsimV2runtime_par.so)
dlopen error: /usr/gapps/visit/2.1.1/linux-x86_64/lib/libengine_par.so: undefined symbol: ICET_STRATEGY_REDUCE

-----------------------REDMINE MIGRATION-----------------------
This ticket was migrated from Redmine. The following information
could not be accurately captured in the new ticket:

Original author: Brad Whitlock
Original creation: 12/28/2010 01:49 pm
Original update: 01/14/2011 01:34 am
Ticket number: 525
